cpp/src/strings/combine.cu

```diff
@@ -85,12 +82,12 @@ std::unique_ptr<column> concatenate( table_view const& strings_columns,
             // for this row (idx), iterate over each column and add up the bytes
             bool null_element = thrust::any_of( thrust::seq, d_table.begin(), d_table.end(),
                 [row_idx] (auto const& d_column) { return d_column.is_null(row_idx);});
-            if( null_element && d_narep.is_null() )
+            if( null_element && !d_narep.is_valid() )
```

Comment:
Curious why you've changed usages of `is_null()` to instead use `!is_valid()`

I think `is_null()` is easier to read than spotting the `!` at the beginning.

Reply:
The `d_narep` variable her is now a `string_scalar` instead of a `string_view`. There is no `is_null()` method on the `scalar` class.

Reply:
Huh, I didn't know that. I figured it was like `column_view` and had both. Understood.